pkg/client/status_reporter.go

```diff
@@ -136,7 +136,7 @@ func (r *StatusReporter) SetFailed(statusErr error, reason string) error {
 	conditions.setCondition(v1.OperatorAvailable, v1.ConditionFalse, "", "", time)
 	conditions.setCondition(v1.OperatorProgressing, v1.ConditionFalse, "", "", time)
 	conditions.setCondition(v1.OperatorDegraded, v1.ConditionTrue, fmt.Sprintf("Failed to rollout the stack. Error: %v", statusErr), reason, time)
-	conditions.setCondition(v1.OperatorUpgradeable, v1.ConditionFalse,
+	conditions.setCondition(v1.OperatorUpgradeable, v1.ConditionTrue,
```

Comment:
This is for when we call `SetFailed `, see description of this PR why I changed this as well. But could use double checking of my logic here :)